apps/web/package.json

```diff
@@ -9,56 +9,60 @@
     "dev": "pnpm run db:start && pnpm run dev:next",
     "dev:debug": "NODE_OPTIONS='--inspect' pnpm run dev:next",
     "build": "next build",
-    "start": "next start",
+    "start": "ts-node src/index.ts",
     "prettier": "prettier --write \"**/*.{ts,tsx,md}\"",
     "tc": "tsc --noEmit",
     "lint": "eslint . --max-warnings 0",
-    "db:generate": "drizzle-kit generate",
-    "db:generate:custom": "pnpm run db:generate --custom",
-    "db:migrate": "drizzle-kit migrate --config=./drizzle.config.ts",
-    "db:migrate:test": "NODE_ENV=test pnpm run db:migrate",
-    "db:drop": "drizzle-kit drop",
-    "db:check": "drizzle-kit check",
-    "db:studio": "drizzle-kit studio --verbose"
+    "db:generate:migration": "pnpm run typeorm migration:generate -d ./src/db/migrations.config.ts",
+    "db:create:migration": "pnpm run typeorm migration:create",
+    "db:up": "pnpm run typeorm migration:run",
+    "db:down": "pnpm run typeorm migration:revert",
+    "typeorm": "tsx --env-file .env ./node_modules/typeorm/cli.js"
```

Comment:
`--env-file` requires [node >= 20.6.0](https://nodejs.org/docs/v20.15.1/api/cli.html#--env-fileconfig) but is awesome. `tsx` runs node under the hood. 

Being able to load env vars like this is awesome 🙌